Makefile

```diff
@@ -2,20 +2,23 @@ PKGS=$(shell go list ./... | grep -v /vendor)
 SHELL_IMAGE=golang:1.8.3
 GIT_SHA=$(shell git rev-parse --verify HEAD)
 VERSION=$(shell cat VERSION)
+PWD=$(shell pwd)
 
 default: bindata compile
+
 compile:
-	@go build -o ${GOPATH}/bin/kubicorn -ldflags "-X github.com/kris-nova/kubicorn/cmd.GitSha=${GIT_SHA} -X github.com/kris-nova/kubicorn/cmd.Version=${VERSION}" main.go
+	go build -o bin/kubicorn -ldflags "-X github.com/kris-nova/kubicorn/cmd.GitSha=${GIT_SHA} -X github.com/kris-nova/kubicorn/cmd.Version=${VERSION}" main.go
```

Comment:
The reason I wanted it to go to `$GOPATH/bin` was so that it would land in my `$PATH`. 

(I have `PATH=$PATH:$GOPATH/bin`)

I am fine with this thought, just giving context